docs/carbon-aware-cli.md

```diff
@@ -117,7 +117,246 @@ command: `.\caw emissions -l eastus -l westus --start-time 2022-07-09T00:00:00Z
 
 output:
 
-```text
+```json
 [{"Location":"eastus","Time":"2022-07-09T00:00:00+00:00","Rating":79.357,"Duration":"12:00:00"},
 {"Location":"westus","Time":"2022-07-09T00:00:00+00:00","Rating":86.91243,"Duration":"12:00:00"}]
-```
\ No newline at end of file
+```
+
+### Command `emissions-forecasts`
+
+#### Description
+
+Forecasted emissions
+
+#### Usage
+
+`caw emissions-forecasts [options]`
+
+#### Options
```

Comment:
Many of these options don't match the implementation.   Let's run the latest `--help` and put the output here

Reply:
Any reason we couldn't just obviate this part of the doc and just tell people to look at help?